developer-docs-site/docs/tutorials/first-coin.md

```diff
@@ -131,8 +136,14 @@ Bob's initial MoonCoin balance: 0.
 Alice mints Bob some of the new coin.
 Bob's updated MoonCoin balance: 100.
 ```
+
+  </TabItem>
+</Tabs>
```

Comment:
This should be up higher right?

Reply:
why? we have multiple Tabs for each language/sdk - check out https://deploy-preview-11189--aptos-developer-docs.netlify.app/tutorials/your-first-coin